.github/workflows/push.yml

```diff
@@ -1,16 +1,27 @@
 name: GitHub Actions
 on: [ push ]
+concurrency: 'ci-${{ github.ref }}'
```

Comment:
Tim padem se ti IMHO nepusti 7.4 a 8.2 job paralelne, ne? To je umyslne?

Reply:
Spustily se paralelně, ale Build action zůstala viset jako Pending. Možná že kvůli tomuhle.